SYCL/ESIMD/regression/unused_load.cpp

```diff
@@ -0,0 +1,65 @@
+//==------ unused_load.cpp  - DPC++ ESIMD on-device regression test --------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// TODO enable on Windows
+// REQUIRES: linux && gpu
```

Comment:
The https://github.com/intel/llvm-test-suite/pull/77/files pr just enables ESIMD tets on Windows. So you can add this test test on Windows as well.